vocabularies/gsq-dataset-themes.ttl

```diff
@@ -0,0 +1,100 @@
+@prefix : <http://linked.data.gov.au/def/gsq-dataset-themes/> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/gsq-dataset-themes> a owl:Ontology .
+
+:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:creator "Geological Survey of Queensland" ;
+    dct:modified "2019-10-02T15:00:47+10:00"^^xsd:dateTime ;
+    dct:source "Developed by the Geological Survey of Queensland." ;
+    skos:definition "A list of broad thematic descriptions of datasets created, received, and held by the Geological Survey of Queensland."@en ;
+    skos:prefLabel "GSQ Dataset Themes"@en .

```

Comment:
Should this be "GSQ dataset themes"?

Reply:
Should it? We do mention Queensland, DNRME, GSQ etc.  but we could replace these with generic 'regulatory body' type terms.

Reply:
It was the capitalisation - not changing it to any other term. I should have been clearer :)

Reply:
doh! fixed. BUT GSQ kept capitals as it is an abbreviated set of initials